tests/pytests/test_geometry_flat.py

```diff
@@ -119,27 +119,20 @@ def testWKTIngestError(env):
   conn = getConnectionByEnv(env)
   env.expect('FT.CREATE idx ON JSON SCHEMA $.geom AS geom GEOSHAPE $.name as name TEXT').ok()
 
-  # IMPORTANT (TODO: workaround?)
-  # We cannot guarantee what will be the last error in cluster, because it depends on the order of replies from the shards
-  # and on what shard indexes what document. For now, we will just check that there is an error.
-
```

Comment:
It's so much better to see this in red...
But I am concerned that the only test that has more than one failure (+is checking the coordinator's new logic in info) doesn't run with sanitizer. Can we just make sure that it is covered in another test?